README.md

```diff
@@ -184,6 +183,24 @@ Sometimes, multi-valued fields may contain missing values; in this case, the `wi
 ### Auxiliary files
 `vembrane` supports additional files, such as lists of genes or ids with the `--aux NAME=path/to/file` option. The file should contain one item per line and is parsed as a set. For example `vembrane filter --aux genes=genes.txt "ANN['SYMBOL'] in AUX['genes']" variants.vcf` will keep only records where the annotated symbol is in the set specified in `genes.txt`.
 
+### Ontologies
+`vembrane` supports ontologies in OBO format. The ontology is loaded into memory and can be accessed in the filter expression via the `SO` symbol. This enables filtering based on relationships between ontology terms. 
+For example, `vembrane filter --ontology so.obo "SO.any_is_a(ANN['Consequence'], "intron_variant")"` will keep only records where at least one of the consequences is an intron variant *or a subtype thereof*.
+If no ontology is provided, the built-in ontology from sequenceontology.org (date: 2024-06-06) is loaded automatically if the `SO` symbol is accessed.
+
+The following functions are available for ontologies:
+- `SO.is_a(term: str, parent: str) -> bool`: Check if there is a path from `term` to `parent`.
```

Comment:
I agree with the one sentence per line sentiment, and also with the wording changes that make it a bit more clear what this function (semantically) implies.
perhaps we can replace "or a subtype of it" with "or inherits from it" or something, to imply it can also be a subtype of a subtype of a subtype …